corehq/messaging/scheduling/views.py

```diff
@@ -1085,3 +1092,48 @@ def post(self, request, *args, **kwargs):
             msg[0](request, msg[1])
 
         return self.get(request, *args, **kwargs)
+
+
+@login_and_domain_required
```

Comment:
is there a permissions decorator that would be appropriate here?  Can you also add a feature flag check?

Reply:
https://github.com/dimagi/commcare-hq/pull/33798/commits/19165c39fc06697c1c9cc85ff3a64a2e43b49a1e